study.md

```diff
@@ -87,13 +87,13 @@ to find information. Additionally, many gems have official 'rdoc' documentation;
 there's a searchable web interface for that documentation at
 **[RubyDoc.info](http://rubydoc.info)**.
 
-Depending on the gem, there may not necessarily be a lot of documentation available, but if the GitHub project has a wiki set up, that can be a good place to find information. Additionally, many gems have official 'rdoc' documentation; there's a searchable web interface for that documentation at **[RubyDoc.info](http://rubydoc.info)**.
```

Comment:
If you check the "Files changed" tab, then it shows the modified block of text above it rather than below it.  I rewrote a lot of the text so that most of the file has 80 characters per line and it seems that this might be an artifact from using patch mode and splitting the changes into hunks when adding the file.